pywinauto/controls/common_controls.py

```diff
@@ -3388,6 +3388,58 @@ def set_view(self, viewType):
         if res == 0:
             raise RuntimeError('Failed to set view in Calendar')
 
+    #----------------------------------------------------------------
+    def set_day_states(self, month_states):
+        """Sets the day states for all months that are currently visible within a month calendar control"""
+        remote_mem = RemoteMemoryBlock(self)
+        day_states = (ctypes.c_uint32 * len(month_states))(*month_states)
+
+        remote_mem.Write(day_states)
+        res = self.send_message(win32defines.MCM_SETDAYSTATE, len(day_states), remote_mem)
+        del remote_mem
+
+        if res == 0:
+            raise RuntimeError('Failed to set the day states in Calendar')
+
+        return res
+
+    #----------------------------------------------------------------
+    def minimize_calendar_rectangle(self, left, top, right, bottom):

```

Comment:
I would call it ```calc_min_rectangle``` (as I understand there is no 'calendar' in other methods names)